lib/workers/repository/process/index.ts

```diff
@@ -84,6 +85,9 @@ export async function extractDependencies(
     logger.debug({ baseBranches: config.baseBranches }, 'baseBranches');
     const extracted: Record<string, Record<string, PackageFile[]>> = {};
     for (const baseBranch of config.baseBranches) {
+      if (!baseBranchesEqualsDefault(config)) {
+        addMeta({ baseBranch });
+      }
```

Comment:
```suggestion
       addMeta({ baseBranch });
```

@rarkins this should be enough, as this code only executes when there is a base branch config.

Reply:
https://github.com/renovatebot/renovate/pull/17027#discussion_r942859916

thats is if we decide to log regardless if baseBranches is basically the default branch only 😀

Reply:
yes, but that's a user decision, no default renovate config or preset. would simplify code.

Reply:
so it is decided then, if baseBranch is set, we log it as part of the metadata regardless whats inside.
ill remove unneeded code than.